README.md

```diff
@@ -66,13 +66,22 @@ brew install gcc | brew link gcc
 
 ```python
 import cornac
-from cornac.eval_methods import RatioSplit
+from cornac.eval_methods import RatioSplit, PropensityStratifiedEvaluation
 from cornac.models import MF, PMF, BPR
 from cornac.metrics import MAE, RMSE, Precision, Recall, NDCG, AUC, MAP
 
 # load the built-in MovieLens 100K and split the data based on ratio
 ml_100k = cornac.datasets.movielens.load_feedback()
+
+# define an evaluation method
+# random train/test split evaluation
 rs = RatioSplit(data=ml_100k, test_size=0.2, rating_threshold=4.0, seed=123)
+# or stratified evaluation
```

Comment:
Let's not change this for now. We would like to keep the very first example as simple as possible for beginners. More advanced features should be presented with examples and more detailed tutorials.